templates/dart/lib/src/chunked_upload_io.dart.twig

```diff
@@ -32,14 +34,28 @@ Future<Response> chunkedUpload({
       headers: headers,
     );
   }
-  // read chunk and upload each chunk
+
   var offset = 0;
+  if(idParamName.isNotEmpty && params[idParamName] != 'unique()') {
```

Comment:
What if they provide `idParamName`, and also provide `id=unique()`? Should we throw error that this combination is not allowed?

Reply:
Nope, idParamName is autogenerated from our spec, not provided by user.